lib/ncdata/xarray.py

```diff
@@ -21,6 +22,16 @@
 from . import NcAttribute, NcData, NcDimension, NcVariable
 
 
+def _raise_warning(var):
+    """Raise a warnings.warning if variable data not lazy."""
+    warn_msg = (
+        f"Variable {var} has fully realized "
```

Comment:
```suggestion
        f"Variable {var.name}{var.dims} has fully realized "
```

I think this would make the messages a lot less annoying!